jerry-core/ecma/builtin-objects/ecma-builtin-helpers-date.cpp

```diff
@@ -1163,7 +1137,12 @@ ecma_date_value_to_utc_string (ecma_number_t datetime_number) /**< datetime */
 } /* ecma_date_value_to_utc_string */
 
 /**
- * Common function to create a time zone specific string from a numeric value.
+ * Length of string created by ecma_date_value_to_iso_string
+ */
+#define ECMA_DATE_VALUE_TO_ISO_STRING_LENGTH 24
```

Comment:
Is this constant local to `ecma_date_value_to_iso_string` function? If it is so, could you, please, move it to the function as a typed constant declaration?